content/en/user-manual/animation/anim-state-graph-assets.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Animstategraph Assets
+template: usermanual-page.tmpl.html
+position: 3
+---
+
+Animstategraph assets are used to organise a set of different animation states, which are all the various ways in which a model might animation. It can be used to define each of these animation states, determine when each state should play and how states transition and therefore blend between one another.. Animstategraph assets do not store or link to any real animation assets themselves, but rather act as a template for how animation assets should be organised. Actual animation assets are linked to the animstategraphs animation states through the [Anim Component](/en/user-manual/packs/components/anim/).
```

Comment:
```suggestion
Animstategraph assets are used to organize a set of different animation states, which are all the various ways in which a model might animate. It can be used to define each of these animation states, determine when each state should play and how states transition and therefore blend between one another. Animstategraph assets do not store or link to any real animation assets themselves, but rather act as a template for how animation assets should be organized. Actual animation assets are linked to the animstategraphs animation states through the [Anim Component](/en/user-manual/packs/components/anim/).
```